requirements.txt

```diff
@@ -3,3 +3,4 @@ matplotlib
 pillow
 sphinx
 nose
+nbformat
```

Comment:
> I started with nbformat, because that is the library for the notebook
> file and it provides their utilities. Now, I just used python standard
> json library.

Good thinking! That was definitely the best way to get quick turn
around times and short cycles.